lib/plugins/static.js

```diff
@@ -44,8 +44,13 @@ function serveStatic(options) {
 
     function serveFileFromStats(file, err, stats, isGzip, req, res, next) {
         if (err) {
-            next(new ResourceNotFoundError(err, '%s', req.path()));
-            return;
+            var defaultFile = path.join(opts.directory, opts.default);
```

Comment:
Hey @JoeDoyle23 Thanks for this submission, and apologies for the late reply. Would you mind adding a unit test for this behaviour? We should be good to merge after that.